packages/utils/package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@trezor/utils",
-    "version": "1.0.0",
+    "version": "1.0.1",
```

Comment:
What is the benefit from publishing @trezor/utils to npm? So it could be used in trezor-connect? (I'm just asking to understand, I'm not suggesting not to do it.)

Reply:
thanks for the explanation